lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMMAv2.cpp

```diff
@@ -559,9 +559,6 @@ Value loadArg(ConversionPatternRewriter &rewriter, Location loc, Value tensor,
 
   SmallVector<Value> multiDimWarpId =
       delinearize(rewriter, loc, warp, warpsPerCTA, order);
-  unsigned lastAxis = order[order.size() - 1];
```

Comment:
@zahimoud Does it match your understanding?

Reply:
Yes, that's what I had in mind. Thanks